crates/router/src/core/payments/operations/payment_reject.rs

```diff
@@ -264,6 +265,16 @@ impl<F: Clone> UpdateTracker<F, PaymentData<F>, PaymentsCancelRequest> for Payme
             )
             .await
             .to_not_found_response(errors::ApiErrorResponse::PaymentNotFound)?;
+        let error_code = payment_data.payment_attempt.error_code;
+        let error_message = payment_data.payment_attempt.error_message;
```

Comment:
```suggestion
        let error_message = payment_data.payment_attempt.error_message.clone();
```

Here as well. CI checks are failing.